lib/diffux_ci/server.rb

```diff
@@ -38,6 +38,27 @@ def h(text)
       end
     end
 
+    get '/*' do
+      config = DiffuxCI::Utils.config
+      current_dir = '/Users/serena_wang/Documents/diffux_ci/lib/diffux_ci'
+      File.open(File.join(current_dir,'output.txt'), 'w') do |f|
+        f.write(Dir.pwd)
+      end
+      file = params[:splat].first
+      if File.exist?(file)
+        send_file file
+      elsif File.exist?(File.join(Dir.pwd, file))
+        send_file File.join(Dir.pwd, file)
+      else
```

Comment:
Do you need both the `if` and `elsif` parts of this route? It seems like you might not.